onsstove/onsstove.py

```diff
@@ -50,8 +50,27 @@ def set_postgres(self, db, POSTGRES_USER, POSTGRES_KEY):
         self.conn = psycopg2.connect(database=db, 
                                      user=POSTGRES_USER, 
                                      password=POSTGRES_KEY)
-                                     
-    
+
+    def read_scenario_data(path_to_config: str) -> Dict[str, Any]:
+        """Reads the scenario data into a dictionary
+        """
+        config = {}  # typing: Dict[str, Any]
+        with open(path_to_config, 'r') as csvfile:
+            reader = DictReader(csvfile, delimiter=';')
```

Comment:
We should create an issue to make the delimiter flexible, so this function works both for `,` and `;`.